nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/JdbcCommon.java

```diff
@@ -300,7 +307,7 @@ public static Schema createSchema(final ResultSet rs, String recordName, boolean
                     // Check the precision of the BIGINT. Some databases allow arbitrary precision (> 19), but Avro won't handle that.
                     // If the precision > 19 (or is negative), use a string for the type, otherwise use a long. The object(s) will be converted
                     // to strings as necessary
-                    int precision = meta.getPrecision(i);
+                    int precision = dbAdapter.getSupportsMetaDataColumnGetPrecision()?meta.getPrecision(i):0;
```

Comment:
Looks like we don't need this. I got tired of fighting exceptions one after another and started reading the HIVE JDBC driver code to find unsupported methods. It looks like I misread the code and getPrecision is a supported HIVE JDBC method call.

https://github.com/apache/hive/blob/ff67cdda1c538dc65087878eeba3e165cf3230f4/jdbc/src/java/org/apache/hive/jdbc/HiveResultSetMetaData.java